resources/scripts/ttx_diff.py

```diff
@@ -121,11 +121,23 @@ def ttx(font_file: Path, can_skip: bool):
     return ttx_file
 
 
-# generate a simple text repr for gpos for this font
+# generate a simple text repr for gpos for this font, with retry
 def simple_gpos_output(cargo_manifest_path: Path, font_file: Path, out_path: Path, can_skip: bool):
+    NUM_RETRIES = 5
+    for i in range(NUM_RETRIES+1):
+        try:
+           return simple_gpos_output_impl(cargo_manifest_path, font_file, out_path, can_skip)
+        except subprocess.CalledProcessError as e:
+            if i == NUM_RETRIES:
```

Comment:
I know it shouldn't happen but ... >= perhaps

Also I think you should wait if you failed, maybe a random period from 100 to 2500 ms or something of that nature